direct-auth-samples/src/test/java/info/seleniumcucumber/userStepDefinitions/SocialAuth.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2021-Present Okta, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package info.seleniumcucumber.userStepDefinitions;
+
+import cucumber.api.java.en.And;
+import cucumber.api.java.en.Given;
+import cucumber.api.java.en.Then;
+import env.CucumberRoot;
+import env.DriverUtil;
+import org.junit.Assert;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import pages.FacebookLoginPage;
+import pages.LoginPage;
+import pages.RootPage;
+
+public class SocialAuth extends CucumberRoot {
+
+    protected WebDriver driver = DriverUtil.getDefaultDriver();
+    private RootPage rootPage = new RootPage(driver);
+    private LoginPage loginPage = new LoginPage(driver);
+    private FacebookLoginPage facebookLoginPage = new FacebookLoginPage(driver);
+
+    @Given("^she clicks the \"Login with Facebook\" button$")
+    public void clicks_facebook_login() {
+        loginPage.sleep();
+        Assert.assertTrue(loginPage.facebookLoginButton.isDisplayed());
+        loginPage.facebookLoginButton.click();
+    }
+
+    @And("^logs in to Facebook$")
+    public void login_facebook() {
+        facebookLoginPage.sleep();
+
+        Assert.assertTrue(facebookLoginPage.emailInput.isDisplayed());
+        facebookLoginPage.emailInput.click();
+        facebookLoginPage.emailInput.sendKeys(USERNAME_FACEBOOK);
+
+        Assert.assertTrue(facebookLoginPage.passwordInput.isDisplayed());
+        facebookLoginPage.passwordInput.click();
+        facebookLoginPage.passwordInput.sendKeys(PASSWORD_FACEBOOK);
+
+        Assert.assertTrue(facebookLoginPage.loginButton.isDisplayed());
+        facebookLoginPage.loginButton.click();
+    }
+
+    @Then("^the Root Page shows links to the Entry Points$")
+    public void theRootPageShowsLinksToTheEntryPoints() {
```

Comment:
nit: If we're planning to use `snakeCase` for all method names, then we should rename all earlier methods with `camel_case` to `snakeCase`. We can do it in a later PR otherwise, for consistency.

Reply:
@vijetmahabaleshwar-okta 
FYI: `snake_case` and `camelCase` :)

Reply:
All method names updated to follow naming consistency (snake_case)